mrmustard/physics/ansatze.py

```diff
@@ -787,6 +787,84 @@ def __call__(self, z: Batch[Vector]) -> Scalar:
             )  # (b_arg)
         return val
 
+    def call_none(self, z: Batch[Vector]) -> DiffOpPolyExpAnsatz:
+        r"""
+        Updates the ansatz from calling the ansatz on some wires, while leaving the rest as None.
+        Note that the batch of the triple and argument in this method is handled parwise, unlike the regular call where the batch over the triple is a superposition.
```

Comment:
what do you mean? can you expand a bit?

Reply:
I added a bit more, let me know if it is still not clear.